go/libraries/doltcore/sqle/history_table.go

```diff
@@ -41,6 +42,12 @@ const (
 
 	// CommitDateCol is the name of the column containing the commit date in the result set
 	CommitDateCol = "commit_date"
+
+	// history tags implement new projection interface, they should not correspond to real tags
+	// and are private to the history table
+	commitHashTag uint64 = iota
+	committerTag
+	commitDateTag
```

Comment:
we have tags reserved tags for this history table here: https://github.com/dolthub/dolt/blob/efdf426a664b235f53704ffd80a982ab6c11d535/go/libraries/doltcore/schema/reserved_tags.go#L42